docs/.npmrc

```diff
@@ -0,0 +1,2 @@
+node-linker=hoisted
+dedupe-peer-dependents=false
```

Comment:
Based on several gh issues in `pnpm` repo.
Tried with both `dedupe-peer-dependents=false` and `dedupe-peer-dependents=true`
For more context https://pnpm.io/npmrc & https://github.com/pnpm/pnpm/issues/6620#issuecomment-1574486773